### DISCUSSION TITLE
Can it support multiple input and output?

### DISCUSSION THREAD

**Discussion by hanrea** [bcakmakoglu/vue-flow#1392] [Q&A] [answered]
In this [Example](https://vueflow.dev/examples/nodes/)   ，I am trying to add two handles in the same location，They have all been rendered, but their positions are the same. Can they be sorted in order?  Or there are plans to support input and output similar to litegraph. js

**Comment by bcakmakoglu** [comment, marked as answer]
Use CSS to position your handles, there's nothing else to it.
Handles are simple `<div>` elements that use `position: absolute` for positioning, meaning the closest wrapping element with `position: relative` will be what the handle aligns itself with.

```vue
  <Handle id="a" type="source" :position="Position.Right" :style="{ top: '10px' }" />
  <Handle id="b" type="source" :position="Position.Right" :style="{ bottom: '10px', top: 'auto' }" />
```
